core/analysis/collation_token_stream.hpp

```diff
@@ -26,6 +26,8 @@
 #include "analyzers.hpp"
 #include "token_attributes.hpp"
 #include "utils/frozen_attributes.hpp"
+#include "unicode/locid.h"
```

Comment:
this header has to be placed before iresearch headers as per guidelines, also use <> instead of quotes